### PR TITLE
V1.2 User Guide

### PR DESCRIPTION
**Pull request by SeeuSim**
A base skeleton highlighting the features and how to use them.

For now, the following operations are provided:
- `Add`
- `View`
- `List`
- `Delete`
- `Exit`

The following functionality is provided:
- Storage after closing (as a JSON file)
- Import of data (via a JSON file, if format is valid)

More features will be released in the next changelog.

### CODE REVIEW

**Comment by ZHTang29** [comment]
Good job on doing the standardisations and formatting for all the different commands!

However for Line 47-50, try to standardise the use caps for the names of the parameters (i.e ‘t/TIME’ and ‘p/PEOPLE_SERVED’.

Otherwise alls good!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,191 +3,214 @@ layout: page
...
 
-Shows a list of all persons in the address book.
+### Adding a Rizz-ipe: `add`
+
+Come up with a new innovative recipe and want to store it for future reference,
+and want to classify it by ingredients? Simply run the `add` command!
+
+Format: 
+```text
+add n/RIZZIPE_NAME \
```
Comment (ZHTang29) [inline docs/UserGuide.md:47]: Amend to ensure standardisation of the parameter names in caps, (i.e ‘t/TIME’, ‘p/PEOPLE_SERVED’).

**Review by ZHTang29 — APPROVED** [review]
Good job otherwise on the other standardisations and formatting!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,191 +3,227 @@ layout: page
...
+- Parameters can be in any order.<br>
+  e.g. if the command specifies `n/NAME p/PERSONS_SERVED`, `p/PERSONS_SERVED n/NAME` is also acceptable.
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+- If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
+  e.g. if you specify `p/3-4 p/5-6`, only `p/5-6` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+- Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
```
Comment (JamesLiuZX) [inline docs/UserGuide.md:45]: To be obsoleted after Seeu Sim's refactoring of add command.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,191 +3,227 @@ layout: page
...
+Come up with a new innovative recipe and want to store it for future reference,
+and want to classify it by ingredients? Simply run the `add` command!
+
+Format: 
+```text
+add n/recipe_NAME \ 
+    t/time p/people served [g/tag1 g/tag2...] \
+    [i/INGREDIENT1 -q QUANTITY1, i/INGREDIENT2 -q QUANTITY2...] \
+    [s-1/STEP 1 s-2/STEP 2...`
+```
```
Comment (JamesLiuZX) [inline docs/UserGuide.md:62]: Same as above.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -3,191 +3,227 @@ layout: page
...
-Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+Example of usage:
+
+`delete 1`
+
+Expected output:
+```
+Deleted dish: Egg Fried Rice.
```
Comment (JamesLiuZX) [inline docs/UserGuide.md:175]: "dish" can be changed to recipe.

File: docs/UserGuide.md (hunk 2 above)
Comment (SeeuSim) [reply docs/UserGuide.md]: Left some in, may be useful for other commands

File: docs/UserGuide.md (hunk 1 above)
Comment (SeeuSim) [reply docs/UserGuide.md]: done

**Review by JamesLiuZX — APPROVED** [review]

**Pull request merged**